Python-Function-MIN.md

```diff
@@ -0,0 +1,36 @@
+# Python min(x)
+
+`min()` is a built-in function in Python 3. It returns the smallest item in an iterable or the smallest of two or more arguments.
+
+## Arguments
+This function takes 2 or more numbers or any kind of iterable as an argument. While giving an iterable as an argument we must make sure 
+that all the elements in the iterable are of same type. This means that we cannot pass a list which has both string and integer values 
+stored in it.
+
+## Return Value
+
+The smallest item in the iterable is returned. If two or more positional arguments are provided, the smallest of the positional arguments 
+is returned. If the iterable is empty and default is not provided, a ValueError is raised.
```

Comment:
@atjonathan I checked the page in its rendered form - the break is not coming up there.